advocacy_docs/partner_docs/CommVaultGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+Commvault enables businesses to proactively simplify and manage the complexity of their continuously evolving and growing data environments, whether your data is on-premises or in the cloud.
+
+Commvault PostgreSQL/EDB Postgres Advanced Server (EPAS) iDataAgent provides the flexibility to backup the PostgreSQL/EDB Postgres Advanced Server (EPAS) databases in different modes and restore them in minimal time. Whether you need to perform a full or log backup or restore of the entire PostgreSQL/EDB Postgres Advanced Server (EPAS) server or individual databases or archive logs at any point of time, you have full control.
```

Comment:
We need to use the product name so should be.
Commvault PostgreSQL iDataAgent provides.  In the rest of the paragraph Advanced Server fits, as we are backing that up.